test/TestHelper.cpp

```diff
@@ -105,10 +108,24 @@ ImportTest::ImportTest(json_spirit::mObject& _o, bool isFiller):
 	if (!isFiller)
 	{
 		importState(_o["post"].get_obj(), m_statePost);
-		m_environment.sub.logs = importLog(_o["logs"].get_array());
+		cnote << "ImportTest(json_spirit::mObject& _o, bool isFiller) Environment logs check ignored!";
+		//m_environment.sub.logs = importLog(_o["logs"].get_array());
 	}
 }
 
+bytes ImportTest::executeTest()
```

Comment:
it should not be part of the importer (who's sole purpose is to import the test json into the State objects) ot execute tests.